Amplify/DevMenu/View/LogViewer.swift

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct LogViewer: View {
+
+    @State private var searchText: String = ""
+    @State private var filter: FilterType = .all
+    private let screenTitle = "Log Viewer"
+    private let searchHint = "Search logs..."
+    private let noLogsAvailable = "No logs available"
+
+    private var logHistory: [LogEntryItem] = LogEntryHelper.getLogHistory()
+
+    enum FilterType: String, CaseIterable, Identifiable {
+        case all
+        case error
+        case warn
+        case info
+        case debug
+        case verbose
+
+        var id: String { rawValue }
+    }
+
+    var filteredList: [LogEntryItem] {
+        switch filter {
+        case .all:
+            return logHistory
+        case .error:
+            return logHistory.filter {$0.logLevel == LogLevel.error}
+        case .warn:
+            return logHistory.filter {$0.logLevel == LogLevel.warn}
+        case .info:
+            return logHistory.filter {$0.logLevel == LogLevel.info}
+        case .debug:
+            return logHistory.filter {$0.logLevel == LogLevel.debug}
+        case .verbose:
+            return logHistory.filter {$0.logLevel == LogLevel.verbose}
+        }
+    }
+
+    var body: some View {
+        VStack {
+            if logHistory.isEmpty {
+                Spacer()
+                Text(noLogsAvailable)
+                Spacer()
+            } else {
+                SearchBar(text: $searchText, searchHint: searchHint)
+                Picker("Filter", selection: $filter) {
+                    ForEach(FilterType.allCases) { filter in
+                        Text(filter.rawValue.capitalized).tag(filter)
+                    }
+                }.pickerStyle(SegmentedPickerStyle())
```

Comment:
nit: horizontal padding so the search bar and the picker are aligned with the view title and the list view items (maybe group them in a `VStack` and add the padding there?).